decompiler/backend/variabledeclarations.py

```diff
@@ -126,4 +127,8 @@ def get_initial_value(variable: GlobalVariable) -> str:
             return str(variable.initial_value.value)
         if isinstance(variable.initial_value, bytes):
             return str(convert_bytes(variable.initial_value, variable.type))
+        if isinstance(operation:=variable.initial_value, Operation):
+            for requirement in operation.requirements:
+                if isinstance(requirement, GlobalVariable) and requirement.ssa_label is not None:
+                    requirement.ssa_label = None
```

Comment:
As far as I see there is no other cod snippet that handles rhs of the declaration (aka. initial value) in case rhs  is an expression, not a single variable or constant, e.g.:
```
int glob_a#0 = 10;
int* glob_b#0 = &glob_a#0 or int* glob#b = a#0 + c#0
```
In case you have other suggestions which part of code should handle ssa labels removal from variables involved in initial values of global variables, I would appreciate that.

Reply:
Simplified a bit.